osu.Game/Overlays/Profile/RankChart.cs

```diff
@@ -24,6 +23,7 @@ public class RankChart : Container
         private readonly RankChartLineGraph graph;
 
         private readonly int[] ranks;
+        private readonly int rankedDays;

```

Comment:
not sure if we really need to make this a field rather than just accessing `.Length` each time.